code/modules/atmospherics/FEA_gas_mixture.dm

```diff
@@ -124,15 +125,25 @@ What are the archived variables for?
 			// has already been asserted above instead of utilizing get_trace_gas_by_type()
 			var/datum/gas/oxygen_agent_b/trace_gas = src.trace_gas_refs[/datum/gas/oxygen_agent_b/]
 			if(trace_gas?.moles > MINIMUM_REACT_QUANTITY )
-				reaction_rate = min(src.carbon_dioxide*0.75, src.toxins*0.25, trace_gas.moles*0.05)
-				reaction_rate = QUANTIZE(reaction_rate)
+				// nitrogen saturates oxygen beta, preventing it from reacting with plasma.
+				if (src.nitrogen > src.toxins)
+					reaction_rate = min(src.nitrogen / 2, src.oxygen, trace_gas.moles * SLEEPING_GAS_CATALYSIS_RATIO)
+					reaction_rate = QUANTIZE(reaction_rate * 2)
+					src.oxygen -= reaction_rate / 2
+					src.nitrogen -= reaction_rate
```

Comment:
So you flood with nitrogen which does nothing until it overcomes toxins, which will then stop the molitz from converting... but then large amounts of the gas will convert to N2O.

You are still left with half the nitrogen and a bunch of N2O, is there a missing benefit to the burn I'm missing?  I was originally assuming a smaller conversion rate to help slow down the state.

Reply:
All of the nitrogen is consumed, the reaction rate is doubled.
Large amounts of Nitrogen is converted because I found that flooding the chamber with N2 while having a less than 100% conversion rate would basically completely choke the burn.
The only benefit is not using up Molitz Beta while you're not around to manage it, pumping N2 is pretty suppressive for the burn overall.